portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -117,18 +137,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     long timestamp = System.currentTimeMillis();
+    SimpleDateFormat dateFormat = new SimpleDateFormat("EEE, MMM d, yyyy h:mm:ss a");
+    String formattedDate = dateFormat.format(new Date(timestamp));
```

Comment:
Generally since this is presentation data and you may want to change the format later (e.g., display in user's time zone, etc), I would opt not to store this. It would also require additional storage space and instead could just be computed at runtime from the timestamp.